bitnami/airflow/templates/scheduler/deployment.yaml

```diff
@@ -127,12 +127,24 @@ spec:
             {{- if .Values.scheduler.extraEnvVars }}
             {{- include "common.tplvalues.render" (dict "value" .Values.scheduler.extraEnvVars "context" $) | nindent 12 }}
             {{- end }}
-          {{- if or .Values.scheduler.extraEnvVarsCM .Values.scheduler.extraEnvVarsSecret .Values.scheduler.extraEnvVarsSecret .Values.extraEnvVarsCM .Values.extraEnvVarsSecret .Values.extraEnvVarsSecrets }}
```

Comment:
Note to self: Double check incidental double up of Values.scheduler.extraEnvVarsSecret, looks like one of these is meant to be secrets